tests/integration/graviton_abi_test.py

```diff
@@ -513,10 +518,12 @@ def py_factorial(n):
     return 1 if n <= 1 else n * py_factorial(n - 1)
 
 
-def test_conditional_factorial():
+@pytest.mark.parametrize("version", [6, 8])
+def test_conditional_factorial(version: int):
     ptdre = PyTealDryRunExecutor(conditional_factorial, pt.Mode.Application)
     inputs = [(n,) for n in range(20)]
-    inspectors = ptdre.dryrun_on_sequence(inputs)
+
+    inspectors = ptdre.dryrun_on_sequence(inputs, compiler_version=version)  # type: ignore
```

Comment:
A lot of the changes in this file have `# type: ignore`. I can't see why that's needed